assets/l10n/app_en.arb

```diff
@@ -194,6 +194,21 @@
       "filename": {"type": "String", "example": "file.txt"}
     }
   },
+  "messageListGroupUnknownUser": "(unknown user)",
+  "@messageListGroupUnknownUser": {
+    "description": "Name placeholder to use for user when we don't know their name."
```

Comment:
```suggestion
    "description": "Name placeholder to use for a user when we don't know their name."
```

The headline-ese "user" without an article sounds like it might mean "*the* user", i.e. the self-user.